webcompat/static/js/lib/comments.js

```diff
@@ -7,7 +7,16 @@ var issues = issues || {};
 issues.CommentsCollection = Backbone.Collection.extend({
   model: issues.Comment,
   url: function() {
-    return '/api/issues/' + issueNumber + '/comments?per_page=100';
+    return '/api/issues/' + issueNumber + '/comments?page=' + this.param;
+  },
+  initialize: function(param) {
+    this.param = param[0].page;
```

Comment:
See my comment below, `param[0].page` should just be `param.page` (unless there's some reason I don't understand why this needs to be an array).